controllers/chat/moveToAnon.js

```diff
@@ -62,26 +79,28 @@ const moveToAnon = async (req, res) => {
     return res.status(404).json(responseError('Old channel id not found'));
   }
 
-  if (!prevTargetUser) {
-    prevTargetUser = {
-      user_id: targetUserId,
-      anon_user_info_color_code: color.code,
-      anon_user_info_color_name: color.color,
-      anon_user_info_emoji_code: emoji.emoji,
-      anon_user_info_emoji_name: emoji.name
-    };
-  }
-  // prevent if anon user info is null
-  if (!prevTargetUser.anon_user_info_color_code || !prevTargetUser.anon_user_info_color_name) {
-    prevTargetUser.anon_user_info_color_code = color.code;
-    prevTargetUser.anon_user_info_color_name = color.color;
-  }
-  if (!prevTargetUser.anon_user_info_emoji_code || !prevTargetUser.anon_user_info_emoji_name) {
-    prevTargetUser.anon_user_info_emoji_code = emoji.emoji;
-    prevTargetUser.anon_user_info_emoji_name = emoji.name;
-  }
-
-  const detailPrevTargetUser = await UsersFunction.findUserById(User, targetUserId);
+  // if (!prevTargetUser) {
+  //   prevTargetUser = {
+  //     user_id: targetUserId,
+  //     anon_user_info_color_code: color.code,
+  //     anon_user_info_color_name: color.color,
+  //     anon_user_info_emoji_code: emoji.emoji,
+  //     anon_user_info_emoji_name: emoji.name
+  //   };
+  // }
+  // // prevent if anon user info is null
+  // if (!prevTargetUser.anon_user_info_color_code || !prevTargetUser.anon_user_info_color_name) {
+  //   prevTargetUser.anon_user_info_color_code = color.code;
+  //   prevTargetUser.anon_user_info_color_name = color.color;
+  // }
+  // if (!prevTargetUser.anon_user_info_emoji_code || !prevTargetUser.anon_user_info_emoji_name) {
+  //   prevTargetUser.anon_user_info_emoji_code = emoji.emoji;
+  //   prevTargetUser.anon_user_info_emoji_name = emoji.name;
+  // }
+
+  // const detailPrevTargetUser = await UsersFunction.findUserById(User, targetUserId);
+  // console.log('DEBUG detailPrevTargetUser XXX => ', detailPrevTargetUser);
+  // console.log('DEBUG prevTargetUser XXX => ', prevTargetUser);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It appears that you've commented out a large block of code here. If this code is no longer needed, it's best to remove it entirely to keep the codebase clean and maintainable.



<!-- This is an auto-generated comment by OSS CodeRabbit -->